std/os.zig

```diff
@@ -145,11 +145,6 @@ fn getRandomBytesDevURandom(buf: []u8) !void {
     const fd = try openC(c"/dev/urandom", O_RDONLY | O_CLOEXEC, 0);
     defer close(fd);
 
-    const st = try fstat(fd);
```

Comment:
I've just spun up an `aarch64` system on Scaleway:

```
Linux test-aarch64-urandom 4.9.93-mainline-rev1 #1 SMP Tue Apr 10 09:54:46 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux
```

Using `file /dev/urandom` returns:

```
/dev/urandom: character special (1/9)
```

Which is as expected. 

I have then spun up an AMD64 system and ran an aarch64 image in QEMU using [these instructions](https://gist.github.com/oznu/ac9efae7c24fd1f37f1d933254587aa4) (though using Ubuntu 18.04 instead of 16.04):

```
Linux ubuntu 4.15.0-64-generic #73-Ubuntu SMP Thu Sep 12 13:18:31 UTC 2019 aarch64 aarch64 aarch64 GNU/Linux
```

Connecting to this and running `file /dev/urandom` results in:

```
/dev/urandom: character special (1/9)
```

As expected.

I'm not sure exactly what the build/test script is doing to test the aarch64 stuff, but it must be doing something weird for `/dev/urandom` to not be a character device...

Reply:
> I have then spun up an AMD64 system and ran an aarch64 image in QEMU using these instructions (though using Ubuntu 18.04 instead of 16.04):

The tests are done using QEMU's userspace emulation instead of the classic full-system approach. All you have to do is cross-compile the test exercising `getRandomBytesDevURandom` and run it with `qemu-aarch64`.

Reply:
I'll give that a go and see what happens then, thanks.